types/map4dsdk/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please do not disable these lint rules unless absolutely necessary. If necessary, please prefer commenting in the code why they must be disabled.